### DISCUSSION TITLE
Allow customization of ChainIcon dimensions

### DISCUSSION THREAD

**Discussion by JamieLottering** [family/connectkit#121] [Ideas]
As a user I want to be able to render the `<ChainIcon />` component in dimensions larger or smaller than the default 24 pixels. `<ChainIcon size={30} />`

**Comment by JamieLottering** [comment]
Opened a PR - https://github.com/family/connectkit/pull/122

**Comment by lochie** [comment]
This feature is now in ConnectKit 1.1.4, thanks!
